kernel/Makefile

```diff
@@ -51,9 +53,12 @@ sign_$(2): build_$(2)
 		 docker tag linuxkit/$(IMAGE):$(1)-$(HASH) linuxkit/$(IMAGE):$(2) && \
 		 DOCKER_CONTENT_TRUST=1 docker push linuxkit/$(IMAGE):$(2))
 
+update-config_$(2): build_$(2)
+	docker run linuxkit/$(IMAGE):$(1)-$(HASH) cat /kernel_config > kernel_config-$(2)
```

Comment:
I was just forcing builds with HASH=foo, so that you can do it before commit, but it shouldn't be hard to do a separate Dockerfile either.